hazelcast/src/test/java/com/hazelcast/spi/impl/operationexecutor/classic/ClassicOperationExecutorTest.java

```diff
@@ -83,4 +85,28 @@ public void run() throws Exception {
             }
         });
     }
+
+    @Test(expected = NullPointerException.class)
+    public void test_runOnAllPartitionThreads_whenTaskNull() {
+        initExecutor();
+        executor.runOnAllPartitionThreads(null);
+    }
+
+    @Test
+    public void test_runOnAllPartitionThreads() {
+        initExecutor();
+
+        int threadCount = executor.getPartitionOperationThreadCount();
+        final CountDownLatch latch = new CountDownLatch(threadCount);
+
+        executor.runOnAllPartitionThreads(new Runnable() {
+            @Override
+            public void run() {
+                latch.countDown();
+            }
+        });
+
+        assertOpenEventually(latch);
```

Comment:
actually the inner class should use a longer timeout then the main test thread? I am not too sure, but it can't hurt. 

from CyclicBarrier contract: "`await()` [...] throws TimeoutException if the specified timeout elapses. *In this case the barrier will be broken.*"